packages/docs/components/landing/Preview.vue

```diff
@@ -1,134 +1,72 @@
-        <!--            <div class="mover"></div>-->
-        <!--          </div>-->
-        <!--        </div>-->
-        <div class="preview__features">
-          <!-- items -->
-          <!--          <div class="item">-->
-          <!--            <div class="item__frame">-->
-          <!--              <img src="/landing/features/nuxt-support.svg" alt="nuxt-support" height="48" width="48" />-->
-          <!--            </div>-->
-          <!--            <h2 class="item__title">{{$t('landing.preview.features.nuxt.title')}}</h2>-->
-          <!--            <router-link class="item__link" :to="`/${$root.$i18n.locale}/getting-started/nuxt-integration`">-->
-          <!--              {{$t('landing.preview.features.nuxt.text')}}-->
-          <!--            </router-link>-->
-          <!--          </div>-->
 
-          <div class="item">
-            <div class="item__frame">
-              <img
-                src="/landing/features/components.svg"
-                alt="Responsive components"
-                height="48"
-                width="48"
-              >
-            </div>
-            <h2 class="item__title">
-              {{ $t('landing.preview.features.components.title') }}
-            </h2>
-            <router-link
-              class="item__link text--primary"
-              :to="`/${locale}/ui-elements/alert`"
-            >
-              {{ $t('landing.preview.features.components.text') }}
-            </router-link>
-          </div>
-
-          <div class="item">
-            <div class="item__frame">
-              <img
-                src="/landing/features/keyboard-navigation.svg"
-                alt="Accessibility"
-                height="48"
-                width="48"
-              >
-            </div>
-            <h2 class="item__title">
-              {{ $t('landing.preview.features.accessibility.title') }}
-            </h2>
-            <router-link
-              class="item__link text--primary"
-              :to="`/${locale}/introduction/accessibility-guide`"
-            >
-              {{ $t('landing.preview.features.accessibility.text') }}
-            </router-link>
-          </div>
-
-          <div class="item">
-            <div class="item__frame">
-              <img
-                src="/landing/features/modern-browsers.svg"
-                alt="Modern browsers"
-                height="48"
-                width="48"
-              >
-            </div>
-            <h2 class="item__title">
-              {{ $t('landing.preview.features.browsers.title') }}
-            </h2>
-            <router-link
-              class="item__link text--primary"
-              :to="`/${locale}/introduction/browser-support`"
-            >
-              {{ $t('landing.preview.features.browsers.text') }}
-            </router-link>
-          </div>
-          <!-- /items -->
+        <div class="preview__features">
+          <va-image
```

Comment:
Wouldn't recommend to use va-image in nuxt app. It still have some SSR issues. Change it to img and ratio wrapper. For me, in both build and dev images are not visible.